src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -107,13 +107,13 @@ class PublicationReleasePage extends Component<Props> {
             <aside className="app-related-items">
               <h3 id="subsection-title">About these statistics</h3>
 
-              <h4 data-testid="publication-page--release-name">
+              <h4 data-testid="release-period">
```

Comment:
make id

Reply:
I can change these to id's, but I'd prefer not to, as data-testid signals that this attribute is being used by tests. Also, react-testing-library uses the data-testid attribute, so if you need a selector on one of these elements for unit tests, there will be duplication.